numpy/lib/mixins.py

```diff
@@ -5,53 +5,54 @@
 
 from numpy.core import umath as um
 
-# None of this module should be exposed in top-level NumPy module.
+# Nothing should be exposed in the top-level NumPy module.
 __all__ = []
 
 
+def _disables_array_ufunc(obj):
```

Comment:
I think this helper function is already logically linked with the class due to it's location in this module.